CASMMON-129 Added Grafana_Dashboards_by_Component.md documentation (#1434)
Cray-HPE/docs-csm: operations/system_management_health/Grafana_Dashboards_by_Component.md

# Grafana Dashboards by Component

> ## _This document show dashboards available per component in Grafana_

Reviewer: This is not really a heading. This should just be body text. 
```suggestion

This document show the dashboards that are available for each component in Grafana.
```

A Grafana dashboard is a powerful open source analytical and visualization tool that consists of multiple individual panels arranged in a grid. The panels interact with configured data sources, including.

- AWS CloudWatch
- Prometheus

> _NOTE: There are some known issues around some of the grafana dashboards, and those are explained in [Troubleshoot_Grafana_Dashboard.md](./Troubleshoot_Grafana_Dashboard.md)._

Reviewer: Grafana should be capitalized.
```suggestion
**NOTE:** There are some known issues around some of the Grafana dashboards; those are explained in [Troubleshoot_Grafana_Dashboard.md](./Troubleshoot_Grafana_Dashboard.md).
```

> ## Dashboards

Reviewer: ```suggestion

## Dashboards
```

##### Ceph Dashboards

> - Ceph - Cluster

Reviewer: Line with Dashboard, since this list is already for Ceph, no need to include it on each item
```suggestion
- Cluster
```

> - Ceph - Host Overview

Reviewer: Similarly, Ceph should be removed from all of the other items in this list.

- Ceph - Host Details
- Ceph - MDS Performance
- Ceph - OSD device details
- Ceph - OSD Overview
- Ceph - Pool Details
- Ceph - Pools Overview
- Ceph - RBD Overview
- Ceph - RGW Instance Detail
- Ceph - RGW Overview

> ##### CoreDNS Dashboard

Reviewer: ```suggestion
### CoreDNS
```

- CoreDNS

> ##### ETCD Dashboard

Reviewer: ```suggestion
### ETCD
```

- etcd
- ETCD Clusters

##### Istio

> - Istio Mesh Dashboard

Reviewer: As with Ceph, Istio should be removed from all of the items in this list.

- Istio Performance Dashboard
- Istio Pilot Dashboard
- Istio Service Dashboard
- Istio Workload Dashboard

> ##### Kea

Reviewer: ```suggestion
### Kea
```

- Kea-dhcp

> ##### Kubernetes

Reviewer: ```suggestion
### Kubernetes
```

> - Kubernetes / API server

Reviewer: As with Ceph, Kubernetes should be removed from all of the items in this list.

-  Compute Resources / Cluster
-  Compute Resources / Namespace (Pods)
-  Compute Resources / Namespace (Workloads)
-  Compute Resources / Node (Pods)
-  Compute Resources / Pod
-  Compute Resources / Workload
-  Controller Manager
-  Kubelet
-  Networking / Cluster
-  Networking / Namespace (Pods)
-  Networking / Namespace (Workload)
-  Networking / Pod
-  Networking / Workload
-  Persistent Volumes
-  Proxy
-  Scheduler
-  StatefulSets

### Nodes

- Exporter Full
- Main

### Postgres

- PostgreSQL Statistics

### Prometheus

- Prometheus

### Use Method

> - Cluster

Reviewer: ```suggestion

- Cluster
```

> ##### Nodes

Reviewer: ```suggestion
### Nodes
```

> ##### Postgres

Reviewer: ```suggestion
### Postgres
```

> ##### Prometheus

Reviewer: ```suggestion
### Prometheus
```

> ##### Use Method

Reviewer: ```suggestion
### Use Method
```

> - USE Method / Cluster

Reviewer: As with Ceph, USE Method should be removed from all of the items in this list.
